property-set/README.md

```diff
@@ -4,7 +4,7 @@ This sample shows how to easily define a Property Set and how to
 access data from it.
 This sample leverages the following policy:
 
-* [Assign Message](https://cloud.google.com/apigee/docs/api-platform/reference/policies/assign-message-policy?hl=en) policy to set the JSON response with values retrieved from a [property set](https://cloud.google.com/apigee/docs/api-platform/cache/property-sets)
+* [Assign Message](https://cloud.google.com/apigee/docs/api-platform/reference/policies/assign-message-policy?hl=en) policy to set the JSON response with values retrieved from a[property set](https://cloud.google.com/apigee/docs/api-platform/cache/property-sets)
```

Comment:
space missing between "a" and "[property set]"

Reply:
Done